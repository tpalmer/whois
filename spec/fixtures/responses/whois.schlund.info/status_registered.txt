
% The data in the WHOIS database of 1&1 Internet AG is provided by
% 1&1 for information purposes, and to assist persons in obtaining
% information about or related to a domain name registration record.
% 1&1 does not guarantee its accuracy. By submitting a WHOIS query,
% you agree that you will use this data only for lawful purposes and that,
% under no circumstances, you will use this data to
%  (1) allow, enable, or otherwise support the transmission by e-mail,
%      telephone, or facsimile of mass, unsolicited, commercial advertising or
%      solicitations to entities other than the data recipient's own existing
%      customers; or
%  (2) enable high volume, automated, electronic processes that send queries or
%      data to the systems of any Registry Operator or ICANN-Accredited registrar,
%      except as reasonably necessary to register domain names or modify existing
%      registrations.
% 1&1 reserves the right to modify these terms at any time.
% By submitting this query, you agree to abide by this policy.

domain:                          1and1.com
created:                         28-Sep-1997
last-changed:                    28-Sep-2011
registration-expiration:         27-Sep-2012

nserver:                         ns-1and1.ui-dns.com 217.160.82.200
nserver:                         ns-1and1.ui-dns.org
nserver:                         ns-1and1.ui-dns.de
nserver:                         ns-1and1.ui-dns.biz

status:                          CLIENT-TRANSFER-PROHIBITED

registrant-firstname:            Andreas
registrant-lastname:             Gauger
registrant-organization:         1&1 Internet Inc.
registrant-street1:              701 Lee Rd.
registrant-street2:              Suite 300
registrant-pcode:                19087
registrant-state:                PA
registrant-city:                 Chesterbrook
registrant-ccode:                US
registrant-phone:                +1.8774612631
registrant-fax:                  +1.6105601501
registrant-email:                hostmaster@oneandone.com

admin-c-firstname:               Andreas
admin-c-lastname:                Gauger
admin-c-organization:            1&1 Internet Inc.
admin-c-street1:                 701 Lee Rd.
admin-c-street2:                 Suite 300
admin-c-pcode:                   19087
admin-c-state:                   PA
admin-c-city:                    Chesterbrook
admin-c-ccode:                   US
admin-c-phone:                   +1.8774612631
admin-c-fax:                     +1.6105601501
admin-c-email:                   hostmaster@oneandone.com

tech-c-firstname:                Andreas
tech-c-lastname:                 Gauger
tech-c-organization:             1&1 Internet Inc.
tech-c-street1:                  701 Lee Rd.
tech-c-street2:                  Suite 300
tech-c-pcode:                    19087
tech-c-state:                    PA
tech-c-city:                     Chesterbrook
tech-c-ccode:                    US
tech-c-phone:                    +1.8774612631
tech-c-fax:                      +1.6105601501
tech-c-email:                    hostmaster@oneandone.com

bill-c-firstname:                Andreas
bill-c-lastname:                 Gauger
bill-c-organization:             1&1 Internet Inc.
bill-c-street1:                  701 Lee Rd.
bill-c-street2:                  Suite 300
bill-c-pcode:                    19087
bill-c-state:                    PA
bill-c-city:                     Chesterbrook
bill-c-ccode:                    US
bill-c-phone:                    +1.8774612631
bill-c-fax:                      +1.6105601501
bill-c-email:                    hostmaster@oneandone.com

% See http://registrar.1und1.info for information about 1&1 Internet AG

