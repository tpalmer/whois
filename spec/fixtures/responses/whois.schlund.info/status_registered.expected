#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("1997-09-28")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2011-09-28")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-09-27")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "1&1"
  should: %s.organization == "1&1 Internet AG"
  should: %s.url          == "http://registrar.1und1.info"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == nil
  should: %s[0].name         == "Andreas Gauger"
  should: %s[0].organization == "1&1 Internet Inc."
  should: %s[0].address      == "701 Lee Rd.\nSuite 300"
  should: %s[0].city         == "Chesterbrook"
  should: %s[0].state        == "PA"
  should: %s[0].zip          == "19087"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.8774612631"
  should: %s[0].fax          == "+1.6105601501"
  should: %s[0].email        == "hostmaster@oneandone.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id           == nil
  should: %s[0].name         == "Andreas Gauger"
  should: %s[0].organization == "1&1 Internet Inc."
  should: %s[0].address      == "701 Lee Rd.\nSuite 300"
  should: %s[0].city         == "Chesterbrook"
  should: %s[0].state        == "PA"
  should: %s[0].zip          == "19087"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.8774612631"
  should: %s[0].fax          == "+1.6105601501"
  should: %s[0].email        == "hostmaster@oneandone.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id           == nil
  should: %s[0].name         == "Andreas Gauger"
  should: %s[0].organization == "1&1 Internet Inc."
  should: %s[0].address      == "701 Lee Rd.\nSuite 300"
  should: %s[0].city         == "Chesterbrook"
  should: %s[0].state        == "PA"
  should: %s[0].zip          == "19087"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.8774612631"
  should: %s[0].fax          == "+1.6105601501"
  should: %s[0].email        == "hostmaster@oneandone.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns-1and1.ui-dns.com"
  should: %s[0].ipv4 == "217.160.82.200"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns-1and1.ui-dns.org"
  should: %s[1].ipv4 == nil
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns-1and1.ui-dns.de"
  should: %s[2].ipv4 == nil
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns-1and1.ui-dns.biz"
  should: %s[3].ipv4 == nil
